src/visit_vtk/lightweight/vtkVisItCellLocator.C

```diff
@@ -162,23 +161,8 @@ vtkVisItCellLocator::~vtkVisItCellLocator()
 
 void vtkVisItCellLocator::FreeSearchStructure()
 {
-  vtkIdList *cellIds;
-  int i;
-  
   if ( this->Tree )
     {
-    for (i=0; i<this->NumberOfOctants; i++)
```

Comment:
This looks like removal of an entire function body. Was this intentional?

Reply:
Yes. I should have commented. Had difficulties trying to fix the warning. I looked at the current vtkCellLocator in VTK and they had the more simplified method.
This class (and a few others) should be updated to correspond more closely with the original, or directly subclassed if possible.